docs/_articles/Power-Production-Distribution-and-Usage-Overview.md

```diff
@@ -0,0 +1,46 @@
+
+  * Photovoltaics: The biggest problem with photovoltaics is they only make low voltage DC current. They are getting better all the time at doing that but that’s still all they do. This low voltage DC current must be converted to a higher voltage AC current to work the appliances in our homes and businesses and this is inefficient. The actual electrical usage of things that could not operate on some other energy source is small in the home and many small businesses. The biggest usage in power comes from heating things up and cooling things down. Electricity must be converted to do these things so efficiency is lost. Another problem with them is the way they store energy. By using batteries toxic and/or expensive materials must be used and many are required to heat or cool a home over several days of bad weather. The last problem is they use rare materials that are not a renewable resource and directly compete with computer chip manufacturing. They also require the same special production facilities to manufacture as computer chips. There are some promising new technologies developing but  they only address some of these shortcomings. This is a good article explaining in much more detail: The ugly side of solar panels
+They are especially a bad idea when used in solar farms see Concentrated solar farms below.
+
+  * Large scale concentrated solar farm: The problems with solar farms, and using photovoltaics only makes these issues worse, is that to create the current level of energy use would require huge areas to be covered with these farms. Some have said well just put them in the desert. They have the attitude of that’s where the sun is and nothing much lives there so it would be perfect. There are several problems with this reasoning. The first is not many people live there either so the energy would have to be delivered over huge distances. There would be a significant loss of efficiency and so would require even more land to be covered to make up for those losses. There would also be the cost of building such a distribution system as the current electric grid is in need of updating as it is. The second reason is not all desert is barren wasteland, there is a very fragile ecosystem that would be disturbed to build these farms. Large concentrated solar farms may have their place in driving energetic industrial processes but should not be considered for supplying houses and businesses that don’t need that kind of power.  Please see the American Physical Society report on Integrating Renewable Electricity on the Grid
```

Comment:
Link needed: American Physical Society report on Integrating Renewable Electricity on the Grid